src/dawn/Compiler/Options.inc

```diff
@@ -32,6 +32,7 @@
 //                  command-line target will be generated which allows to negate the option i.e 
 //
 
+OPT(std::string, OutputFile, "", "output", "o", "Write output to <file>", "<file>", true, false)
```

Comment:
I would move it to after the back end decision since this is the more "natural" flow to read the options. Decide on the back end and then choose the output file. Or what was your reasoning to have it here?

Reply:
moved in last commit